legal_info/chamber_to_area.json

```diff
@@ -0,0 +1,220 @@
+    "JU_TC_011": "civil_law",
+    "JU_TPI_006": "penal_law",
+    "JU_TPI_007": "public_law",
+    "LU_KG_003": "social_law",
+    "LU_KG_004": "public_law",
+    "LU_RR_001": "public_law",
+    "LU_JSD_001": "public_law",
+    "LU_BKD_001": "public_law",
+    "LU_GSD_001": "public_law",
+    "NE_TC_001": "civil_law",
+    "NE_TC_002": "civil_law",
+    "NE_TC_003": "penal_law",
+    "NE_TC_004": "civil_law",
+    "NE_TC_008": "penal_law",
+    "NE_TC_009": "penal_law",
+    "NE_TC_011": "penal_law",
+    "NE_TC_012": "public_law",
+    "NE_TC_013": "public_law",
+    "NE_TC_014": "social_law",
+    "NE_TR_001": "civil_law",
+    "NE_ARA_001": "public_law",
+    "NE_ASL_001": "public_law",
+    "NE_ASS_001": "public_law",
+    "NE_ATS_001": "public_law",
+    "NE_CA_001": "public_law",
+    "NE_CN_001": "public_law",
+    "OW_VG_001": "public_law",
+    "SG_KG_001": "penal_law",
+    "SG_KG_002": "civil_law",
+    "SG_KG_003": "public_law",
+    "SG_HG_001": "civil_law",
+    "SG_ABSK_001": "civil_law",
+    "SG_VG_001": "public_law",
+    "SG_VGN_001": "public_law",
+    "SG_VB_001": "public_law",
+    "SO_OG_001": "penal_law",
+    "SO_OG_002": "penal_law",
+    "SO_OG_003": "penal_law",
+    "SO_OG_004": "civil_law",
+    "SO_OG_005": "civil_law",
+    "SO_OG_006": "penal_law",
+    "SO_VG_001": "public_law",
+    "SO_SK_001": "public_law",
+    "SO_STG_001": "public_law",
+    "SZ_KG_001": "civil_law",
+    "SZ_KG_003": "penal_law",
+    "SZ_KG_004": "penal_law",
+    "TI_PP_001": "penal_law",
+    "TI_TCA_001": "public_law",
+    "TI_TCAS_001": "social_law",
+    "TI_CATI_001": "public_law",
+    "TI_TE_001": "public_law",
+    "TI_TRAC_001": "civil_law",
+    "TI_TRAC_002": "civil_law",
+    "TI_TRAC_003": "civil_law",
+    "TI_TRAC_004": "civil_law",
+    "TI_TRAC_006": "civil_law",
+    "TI_TRAP_001": "penal_law",
+    "TI_TRAP_002": "penal_law",
+    "TI_TPC_001": "penal_law",
+    "TI_CARP_001": "penal_law",
+    "TI_GPC_001": "penal_law",
+    "TI_CRP_001": "penal_law",
+    "TI_GIAR_001": "penal_law",
+    "VD_TC_001": "public_law",
+    "VD_TC_002": "civil_law",
+    "VD_TC_003": "penal_law",
+    "VD_TC_004": "social_law",
+    "VD_TC_005": "public_law",
+    "VD_TC_006": "penal_law",
+    "VD_TC_007": "civil_law",
+    "VD_TC_013": "penal_law",
+    "VD_TC_014": "civil_law",
+    "VD_TC_015": "penal_law",
+    "VD_TC_016": "penal_law",
+    "VD_TC_030": "public_law",
+    "VD_TC_031": "public_law",
+    "VS_AG_001": "civil_law",
+    "VS_SRK_001": "public_law",
+    "ZG_VG_001": "social_law",
+    "ZG_VG_002": "public_law",
+    "ZG_VG_004": "public_law",
+    "ZG_VG_999": "public_law",
+    "ZH_BK_004": "civil_law",
+    "ZH_OG_001": "civil_law",
+    "ZH_OG_002": "penal_law",
+    "ZH_OG_003": "penal_law",
+    "ZH_OG_004": "public_law",
+    "ZH_OG_005": "public_law",
+    "ZH_HG_001": "civil_law",
+    "ZH_VG_001": "public_law",
+    "ZH_SRK_001": "public_law",
+    "ZH_BRK_001": "public_law",
+    "ZH_SVG_001": "social_law"
```

Comment:
how did you determine these?

Reply:
These were extracted from the `legal_info/court_chambers_extended.json` file. 
Code for this can be found in `scrc/utils/chamber_areas_from_law_codes.py`